src/solvers/refinement/string_refinement.cpp

```diff
@@ -773,10 +777,12 @@ static exprt get_array(
   const std::function<exprt(const exprt &)> &super_get,
   const namespacet &ns,
   const std::size_t max_string_length,
-  const exprt &arr,
-  const exprt &size)
+  messaget::mstreamt &stream,
+  const array_string_exprt &arr)
 {
-  exprt arr_val=simplify_expr(get_array(super_get, arr), ns);
+  const auto eom=messaget::eom;
```

Comment:
`const auto &`

Reply:
It's a pointer to free function. Pointers should be used by value not by reference, IMHO.